src/components/Report/DownloadCsvButton.tsx

```diff
@@ -8,17 +8,14 @@ import './styles.css';
 import { useAppDispatch } from '@/hooks/common';
 import { clearLoading, setLoading } from '@/reducers/loaderSlice';
 import { getCoordinationTableRows } from '@/hooks/useCoordinationRows';
-import { IListItem } from '@/interfaces/common';
 import { IPlanningRow } from '@/interfaces/planningInterfaces';
-import { ILocation } from '@/interfaces/locationInterfaces';
 
 interface IDownloadCsvButtonProps {
   type: ReportType;
-  categories: IListItem[];
-  divisions: Array<ILocation>;
   getForcedToFrameData: (year: number, forcedToFrame: boolean) => getForcedToFrameDataType;
   getPlanningData: (year: number) => Promise<IPlanningData>;
   getPlanningRows: (res: IPlanningData) => IPlanningRow[];
+  getCategories: () => Promise<any>;
```

Comment:
The type is a bit harder to add here since that is the type of the original fetch function that I'm using
![Screenshot 2024-04-10 at 12 17 46](https://github.com/City-of-Helsinki/infraohjelmointi-ui/assets/77616473/db43b1b1-88e6-45fa-8e85-b1bbc3e00974)
![Screenshot 2024-04-10 at 12 18 06](https://github.com/City-of-Helsinki/infraohjelmointi-ui/assets/77616473/c257d13f-b598-4c80-aee1-bad862d77f4a)

Reply:
The type could be added for the data that the original fetch function returns like
`export const getProjectCategories = async (): Promise<type here> => {`

Reply:
I added a type for the original function now as well.